### PR TITLE
Add Upload migration

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -142,6 +142,14 @@ Feel free to write to our [Support Center](http://devexpress.com/support/center)
     - [Change the Color of Legend Item Icons](#change-the-color-of-legend-item-icons)
     - [Change the Tooltip Z-Index](#change-the-tooltip-z-index)
     - [Set the Tooltip Background Color](#set-the-tooltip-background-color)
+  - [DxUpload](#dxupload)
+    - [Change Select Button Color](#change-select-button-color)
+    - [Change Color of Labels with File Sizes](#change-color-of-labels-with-file-sizes)
```
Comment (albertov05) [inline readme.md]: What do you mean by "_with_ File Sizes"? For example, do you mean "**based on** File Size"?

File: readme.md [hunk 2]
```diff
@@ -3435,4 +3443,173 @@ In v24.1, use the following CSS rules:
...
+In v24.1, use the following CSS rule:
+
+```css
+.dxbl-upload-select-btn {
+    background-color: blue !important;
+    color: white !important;
+}
+```
+
+#### Change Color of Labels with File Sizes
```
Comment (albertov05) [inline readme.md]: See my comment above.

File: readme.md (hunk 1 above)
Comment (SvetlanaMikheeva) [reply readme.md]: I mean the following text string:
image

Comment (albertov05) [reply readme.md]: I see now. I think we can update it as follows:

```suggestion
    - [Change Color of File Size Labels](#change-color-of-labels-with-file-sizes)
```

File: readme.md (hunk 2 above)
Comment (albertov05) [reply readme.md]:
```suggestion
#### Change Color of File Size Labels
```

File: readme.md (hunk 1 above)
Comment (SvetlanaMikheeva) [reply readme.md]:
```suggestion
    - [Change Color of File Size Labels](#change-color-of-file-size-labels)
```

**Pull request merged**